### PR TITLE
Move the docs to WordPress.org

### PR DESCRIPTION
**Pull request by pento**
Docutron was lovely, but it's time to move the docs across to WordPress.org.

The docs will live here: https://wordpress.org/gutenberg/handbook/

WordPress.org will sync the docs from master every 15 minutes (though syncing is currently disabled, until this PR is merged).

### CODE REVIEW

**Review by codebykat — APPROVED** [review]
I fixed a couple typos, other than that this is looking good. Didn't see any missed references to docutron and everything still builds fine. 👍

File: docs/index.js [hunk 1]
```diff
@@ -1,152 +0,0 @@
...
-	name: 'articles',
-	title: 'Articles',
-	markdown: require( './articles.md' ),
-} );
-
-addStory( {
-	parents: [ 'outreach' ],
-	name: 'meetups',
-	title: 'Meetups',
-	markdown: require( './talks.md' ),
```
Comment (Soean) [inline docs/index.js]: Should be meetups.md

Comment (pento) [reply docs/index.js]: Good catch, thanks @Soean!

**Comment by youknowriad** [comment]
Sad to see `docutron` go away. We will never be able to embed components this way. But I understand this is not in the direct scope of Gutenberg.

I think the menu order is not correct. The previous/next links are broken too.

**Comment by aduth** [comment]
>I think the menu order is not correct.

Might make more sense for `manifest.json` to be defined as an array vs. an object.

**Comment by aduth** [comment]
>I think the menu order is not correct.

Might make more sense for `manifest.json` to be defined as an array vs. an object.

**Comment by pento** [comment]
> Might make more sense for `manifest.json` to be defined as an array vs. an object.

Good idea, I've update the theme and this PR to do exactly that, it gets the menu order from the `manifest.json` array order.

**Pull request merged**